openrr-planner/src/collision/self_collision_checker.rs

```diff
@@ -151,6 +157,11 @@ where
         }
         Ok(())
     }
+
+    // Get the robot model used for collision checking
```

Comment:
Is there any reason why you didn't make this a doc string?

Reply:
Exactly. But I found I made the same mistake for JointPathPlanner::collision_check_robot. I would like to fix them in another PR.